transport/http/handler.go

```diff
@@ -96,8 +121,6 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 		ctx = baggage.NewContextWithHeaders(ctx, headers.Items())
 	}
 
-	start := time.Now()
-
 	// TODO capture and handle panic
 	err = h.Handler.Handle(ctx, httpOptions, treq, newResponseWriter(w))
 
```

Comment:
NB: if error indicates a failure of the handler to process the request, you might want to:
  * set `ext.Error = true` tag
  * `span.logEvent(errorMessage)`